Lib/fontTools/cffLib/specializer.py

```diff
@@ -50,7 +50,10 @@ def programToCommands(program):
 				try:
 					width = stack.pop(0)
 				except IndexError:
-					raise ValueError(stack)
+					if ignoreErrors:
+						width = ''
```

Comment:
hm, ok if you say so.. To me it would be clearer if it skipped appending the command in that case. But nevermind